scripts/build_cudaq.sh

```diff
@@ -119,6 +119,28 @@ else
   fi
 fi
 
+# Build and install AWS SDK if necessary
```

Comment:
Why is this built separately here instead of configured to build as part of the CMake configurations like the other submodules? This is not the right place to build it, since the installer and the python wheels won't use the build_cudaq.sh script.

Reply:
Thanks, didn't realize this. Will move this logic into the CMake config.